src/cmd/src/lib.rs

```diff
@@ -169,8 +169,9 @@ pub async fn process_cmd(
                         .close_session(session.as_ref().unwrap().session_id)
                         .await
                     {
-                        Ok(id) => {
-                            println!("Close Session {}", id);
+                        Ok((sess_info_node, sess_info_client)) => {
+                            println!("Close Session Successfully!\nNode session {:?}\nClient session: {:?}",
```

Comment:
Do not show the debug information to a normal user, you can add a flag or verbose to show it or not